src/library.rs

```diff
@@ -277,6 +277,14 @@ impl TypeId {
     pub fn tid_os_string() -> TypeId {
         TypeId { ns_id: 0, id: 33 }
     }
+
+    pub fn tid_gstrv() -> TypeId {
+        TypeId { ns_id: 0, id: 34 }
+    }
+
+    pub fn tid_utf8_list() -> TypeId {
+        TypeId { ns_id: 0, id: 258 }
+    }
```

Comment:
How these 2 types looks in .gir file?
Property `ScaleButton::get_property_icons(&self) -> Vec<GString>` don't detected by it :(
So maybe its don't have constant id and need be added in Library::new after fundamentals.

Reply:
The icons property has the following type in the gir file:
```
        <array>
          <type name="utf8"/>
        </array>
```

Reply:
I already see `Type::c_array(&mut library, TypeId::tid_utf8(), None, None);` in Library::new()  :/

Reply:
Strange that you get this big number then,
if this really same type, then `Library::new` need assert.
PS. And what about second added type?

Reply:
I don't remember for which module I added `gstrv` support actually. I commented it out and no new error appeard :confused: